frontend/docusaurus.config.cjs

```diff
@@ -60,7 +60,13 @@ const config = {
         },
         items: [
           { to: '/badges', label: 'Badges', position: 'left' },
+          {
+            to: '/docs',
+            label: 'Documentation',
```

Comment:
"Documentation" might not be the best title for this menu item, but I don't have a better idea. Very open to suggestions on this.

Reply:
No brilliant alternatives in mind. We can change it later if anything comes up :)